app/src/test/java/uk/ac/ebi/atlas/experimentpage/cellplot/JsonCellPlotControllerWIT.java

```diff
@@ -90,10 +84,8 @@ void cleanDatabaseTables() {
     @BeforeEach
     void setUp() {
         var experimentAccession = jdbcUtils.fetchRandomPublicExperimentAccession();
-        var experiment = experimentTrader.getPublicExperiment(experimentAccession);
-
-        experimentCrud.updateExperimentPrivate(experimentAccession, true);
-        accessKey = experiment.getAccessKey();
+        jdbcUtils.updatePublicExperimentAccessionToPrivate(experimentAccession);
+        accessKey = jdbcUtils.fetchExperimentAccessKey(experimentAccession);
 
```

Comment:
Hi @alfonsomunozpomer - we are thinking that these lines are                                                                        
```
jdbcUtils.updatePublicExperimentAccessionToPrivate(experimentAccession);
 accessKey = jdbcUtils.fetchExperimentAccessKey(experimentAccession); 
```  
 better than the previous lines as we are not calling the actual codebase to get test data inputs.

I couldn't find any suitable method to replace this` expetedCluster() `with JDBCutils. I could see `fetchKsFromCellGroups`, but expected outcome is a `preferredK`.

Please suggest us if you have other opinion on this, we can discuss.